src/inputs/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryFn } from "@storybook/react";
+import { fireEvent, within } from "@storybook/testing-library";
+import { useState } from "react";
+import { useFilter } from "react-aria";
+import { Css } from "src/Css";
+import { Autocomplete } from "src/inputs/Autocomplete";
+
+export default {
+  component: Autocomplete,
+} as Meta;
+
+export const Example: StoryFn = Template.bind({});
+Example.parameters = { chromatic: { disableSnapshot: true } };
```

Comment:
It means we will not take snapshots of this Story ([docs link](https://www.chromatic.com/docs/ignoring-elements#ignore-stories)). The other story includes the input field **and** the menu open, which encompasses everything we'd really want to snapshot. I'm trying to be more cognizant which Stories we _really_ need visually regression tests against. Since everything in this Story is covered by the `MenuOpen` story, then I deemed this one superfluous.

Reply:
Ah nice! Yeah, sorry I could intuit "it turns off snapshots", I should have better articulated "why?", but you're reasoning makes sense.

Wdyt about an in-source const like `chromatic: skipSnapshotsForCost` or something with an in-source documentation/rationale for "we're skipping this b/c its covered by other stories" and not something like "we're skipping this b/c the snapshot is flakey"?